src/notebook.c

```diff
@@ -545,8 +532,31 @@ static gboolean notebook_tab_bar_click_cb(GtkWidget *widget, GdkEventButton *eve
 }
 
 
+static gboolean notebook_menu_btn_press(GtkWidget *widget, GdkEventButton *event,
+	gpointer user_data)
+{
+	// menu has to persist after calling gtk_menu_popup
+	static GtkWidget *menu = NULL;
+
+	if (menu)
+		gtk_widget_destroy(menu);
+		
+	menu = gtk_menu_new();
+	ui_menu_add_document_items(GTK_MENU(menu), document_get_current(),
+		G_CALLBACK(tab_bar_menu_activate_cb));
+	gtk_menu_popup(GTK_MENU(menu), NULL, NULL, NULL, NULL, event->button, event->time);
+	return TRUE;
+}
+
 void notebook_init(void)
 {
+	GtkWidget *btn = gtk_button_new_with_label("☰");
```

Comment:
I think I've seen in some GNOME apps they use icon button with an icon like `view-list` or `view-more` or maybe some more semantically correct name for same icon. I guess the advantage of the icon is that it looks correct no matter which font or size being used (ex. in my browser here, it looks like an equal symbol).

Reply:
Ok, but do they work on Gtk 2? (More generally, how do we know which strings are guaranteed to have icons with Gtk 3, is there a list of common strings? (I saw they deprecated stock items).

Reply:
BTW haven't tried it, but Gtk 3 has a menu button widget that defaults to a down arrow:
https://developer.gnome.org/gtk3/stable/GtkMenuButton.html

Reply:
@codebrainz I read that "open-menu-symbolic" is the icon that matches the unicode symbol. But that doesn't work on GTK2 with gtk_image_new_from_icon_name.

> in my browser here, it looks like an equal symbol

Seems like a font bug?

Reply:
> I read that "open-menu-symbolic" is the icon ...

Yeah, I think that's the one I was trying to remember. Depending on icon theme I think I've seen various images used such as three vertical dots or a gear as well.

> But that doesn't work on GTK2 with gtk_image_new_from_icon_name

It should work fine if you have this icon installed. If it's not commonly available, we could always ship a fallback icon like we do for the other non-standard ones we use.

> Seems like a font bug?

Nah, it's just because the font size is small so two of the lines are squished together making it look more like an equal sign than a list/burger. This is why an icon might be better, it doesn't depend on the user having their font size set large or having a proper Unicode font installed.

Reply:
> a fallback icon like we do for the other non-standard ones we use.

:hamburger: FTW!!